src/chi/components/popover/popover.scss

```diff
@@ -0,0 +1,155 @@
+@import 'variables';
+@import 'mixins';
+
+$popover-arrow-width: 1rem;
+$popover-arrow-height: 0.5rem;
+$popover-border-width: 0.0625rem;
+
+
+.chi {
+  .m-popover {
+    background-color: set-color(white);
+    border-radius: $border-radius;
+    border: $popover-border-width solid set-color(grey, 30);
+    color: $text-color;
+    display: none;
+    font-family: $font-family;
+    font-size: $text;
+    font-weight: normal;
+    left: 0;
+    line-height: $line-height-small;
+    max-width: 18rem;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    z-index: $zindex-popover;
+
+    &.-animated {
+      transition: opacity .2s, transform .2s;
+    }
+
+    &.-active {
+      display: block;
+      opacity: 1;
+    }
+
+    &.-transitioning {
+      display: block;
+    }
+
+    .a-arrow {
+      position: absolute;
+      display: block;
+      width: $popover-arrow-width;
+      height: $popover-arrow-height;
+      margin: 0 $border-radius;
+
+      &::before,
+      &::after {
+        position: absolute;
+        display: block;
+        content: "";
+        border-color: transparent;
+        border-style: solid;
+      }
+    }
+  }
+
+  .m-popover--top, [x-placement^="top"] {
```

Comment:
@dani-cl-madrid x-placement is affecting dropdowns. It looks like It must be in .m-popover